lib/exclamation/configuration.rb

```diff
@@ -62,9 +62,8 @@ def load_default_yaml(name)
     end
 
     def load_yaml(path)
-      puts "loading file: #{path}"
       YAML.load_file(path)
     end
 
   end
-end
\ No newline at end of file
+end
```

Comment:
Sorry for all the whitespace noise, my vim setup automatically strips trailing whitespace.